<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShironekoTranslateClass</class>
 <widget class="QMainWindow" name="ShironekoTranslateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>754</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Translate 1.0 - by lc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="shironekotranslate.qrc">
    <normaloff>:/ShironekoTranslate/shironeko.ico</normaloff>:/ShironekoTranslate/shironeko.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(237, 243, 255)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>414</width>
      <height>738</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#000</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="doneTextPreview">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="stopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 228);
border-color: rgb(7, 7, 7);</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QPushButton" name="tickButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>Mark Subtitle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="startButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 228);
border-color: rgb(7, 7, 7);</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="resultImage">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>414</width>
      <height>738</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#000</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="comingTextPreview">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>371</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="resultLabelUpClickArea">
    <property name="geometry">
     <rect>
      <x>814</x>
      <y>12</y>
      <width>411</width>
      <height>731</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="shironekotranslate.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>resultLabelUpClickArea</sender>
   <signal>clicked()</signal>
   <receiver>startButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1019</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
